kustomize/move2kube/base/01-sonataflow_m2k.yaml

```diff
@@ -0,0 +1,281 @@
+      name: TransformationError
+      transition:
+        nextState: PrintExitErrorMessage
+      type: inject
+    - branches:
+      - actions:
+        - actionDataFilter:
+            useResults: true
+          functionRef:
+            arguments:
+              message: '${"m2k workflow: " + $WORKFLOW.instanceId + " has finalized
+                with error. Exit message: " + .exitMessage + " -- Context: "  + .error
+                }'
+            invoke: sync
+            refName: systemOut
+          name: printSystemOut
+        name: printSystemOut
+      - actions:
+        - actionDataFilter:
+            useResults: true
+          condition: ((has("notificationsDisabled") and .notificationsDisabled !=
+            null and .notificationsDisabled == true)|not)
+          functionRef:
+            arguments:
+              message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace
+                " + .workspaceId + " and project " + .projectId + " failed with exit
+                message: "+ .exitMessage + "\n\nError: "  + .error'
+              origin: Move2Kube Workflow
+              title: '"Move2Kube workflow " + $WORKFLOW.instanceId + " failed"'
+              topic: Move2Kube Workflow
+            invoke: sync
+            refName: createNotification
+          name: createNotification
+        name: createNotification
+      completionType: allOf
+      end:
+        terminate: true
+      name: PrintExitErrorMessage
+      numCompleted: 0
+      type: parallel
+    - branches:
+      - actions:
+        - actionDataFilter:
+            useResults: true
+          functionRef:
+            arguments:
+              message: '${"m2k workflow: " + $WORKFLOW.instanceId + " has finalized
+                successfully}"'
+            invoke: sync
+            refName: systemOut
+          name: printSystemOut
+        name: printSystemOut
+      - actions:
+        - actionDataFilter:
+            useResults: true
+          condition: ((has("notificationsDisabled") and .notificationsDisabled !=
+            null and .notificationsDisabled == true)|not)
+          functionRef:
+            arguments:
+              message: '"Move2Kube workflow  " + $WORKFLOW.instanceId + " on workspace
+                " + .workspaceId + " and project " + .projectId + " was successful;
+                the output is available in the branch " + .targetBranch + " of your
+                git repository "+ .repositoryURL"'
+              origin: Move2Kube Workflow
+              title: '"Move2Kube workflow " + $WORKFLOW.instanceId + " success"'
+              topic: Move2Kube Workflow
+            invoke: sync
+            refName: createNotification
+          name: createNotification
+        name: createNotification
+      completionType: allOf
+      end:
+        terminate: true
+      name: PrintExitSuccessMessage
+      numCompleted: 0
+      type: parallel
+  podTemplate:
+    container:
+      resources: {}
```

Comment:
isn't the `image` missing?

Reply:
nevermind I did not reached `kustomize/move2kube/overlays/prod/sonataflow-patch.yaml` yet